registry/management/gitops.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Each app should be independent.
For example, in Argo CD UI, the `AppProject` should have an app for `cert-manager`, `external-secrets`, and so on.

This file `gitops.yaml` should only load the Kustomize file, and all other resources should be rendered in Kustomize.